app/views/comments/_questioncomments.html.erb

```diff
@@ -6,7 +6,7 @@
           <p><%= comment.body %></p>
           <div id='delete-comment-button' class='left card-action'>
             <% if current_admin? || current_users_comment?(comment)%>
-              <%= button_to "Delete Comment", comment_path(comment), method: :delete, class: "waves-effect red darken-4 waves-light btn-small" %>
+              <%= button_to "Delete Comment", comment_path(comment), method: :delete, class: "white-text waves-effect red darken-4 btn-small" %>
```

Comment:
Thank you for fixing the color of the text! I wasn't sure how to go about it, and it was driving me batty.